docs/src/markdown-pages/guides/permissions-configuration.md

```diff
@@ -0,0 +1,53 @@
+---
+slug: 'guides/permissions-configuration'
+title: 'Configure Loop Permissions'
+description: "Learn how to set up Loop permissions"
+---
+
+# Permission Configuration
+
+If your Loop requires different permissions for different development environments, you can create special configuration files which can be loaded when running your Loop build.
+
+These configuration files should be named according to this pattern: `ldk-config.env.json`, where `env` can any string that indicates the name of a development environment. For example: `ldk-config.dev.json` or `ldk-config.qa.json`.
+
+Per-environment configs can be activated by setting the `LDK_CONFIG` environment variable when running the Loop transpilation process. For example, running `LDK_CONFIG=dev npm run build` will load the configuration in `ldk-config.dev.json`.
+
+The permissions you declare in the `ldk` configuration section of your Loop's `package.json` are considered the "base" permissions config. For Aptitudes which require specific configuration (namely, the Network and Filesystem Aptitudes), any configuration in the environment specific config file will **replace** any values for these permissions in the base configuration. Running the build with no value for `LDK_CONFIG` set will result in only the base configuration being applied.
```

Comment:
**override** might be a better choice of words since you are still keeping the rest of the base config.

Reply:
Sounds good to me! I'll change this over.